### DISCUSSION TITLE
google-cloud-storage

### DISCUSSION THREAD

**Discussion by rodrigosantistebanmel** [keithrozario/Klayers#437] [Ideas]
hey guys! really grateful with you all for the effort in this Klayers repo, already used this to implement requests library in a function. Not really a developer here but really struggling to use google-cloud-storage library in AWS Lambda, tried a lot of things but unable to do this, do you have any ideas to achieve it?

**Comment by keithrozario** [comment]
If you make a pull request for adding it. I'll approve it.
You'll need for fork this report, and make a change to the config file. Then make a pull request.

Keith
